PullRequestCreated/PullRequestCreated.ts

```diff
@@ -55,8 +55,19 @@ class PullRequestCreated extends OctokitAction {
     }
   }
 
+  private async firstNonSubTask(issues: string[]): Promise<any | null> {
+    console.log('Looking for a non-Sub-task ticket');
```

Comment:
Logging for this process is not here and in the caller site. I would like to have it on a consistent level instead.
It means either:
* Move `Found mentioned issues..` inside `findmentionedIssues`, and move `Parent issue....` here.
* Move `Looking for non-Sub-task ticket` to `newIssueParameters`